faq.md

```diff
@@ -33,6 +35,16 @@ Sigrid CI: Frequently Asked Questions
 
 ## Usage questions
 
+### Do you support pull request integration?
+
+Yes! There are basically two usage scenarios for Sigrid CI, and you would typically use them both.
+
+The first Sigrid CI use case is to *publish* your project's source code to Sigrid, which makes the analysis results accessible on [https://sigrid-says.com](https://sigrid-says.com). You would typically use this for your project's main branch, which can be either `main`, `master`, or something project-specific.
+
+The second use case is to use Sigrid CI for feedback during code reviews on pull requests. In these situations, you would typically *not* publish the results to Sigrid, since the code review is usually been author and reviewer and does not require the entire team to see the results. 
```

Comment:
> since the code review is usually been author and reviewer and does not require the entire team to see the results.
I'm not sure what you mean here. Should "been" be "between"? Then the sentence makes sense to me.

On another note, I would say another important reason is that Sigrid CI compares to the code as published to Sigrid, so in general you would not want to publish code that has not yet been merged into your main branch. Would that be good to add here?

Reply:
We do explain that point https://github.com/Software-Improvement-Group/sigridci/blob/main/faq.md#what-branch-or-commit-does-sigrid-ci-compare-against. I'm hesitant to repeat the entire argument in the platform-specific instructions. As a middle ground, let's include a very brief description in these bullets.

Reply:
Oh, and the "been/between" was indeed a typo :)